splitgpg2/__init__.py

```diff
@@ -199,7 +200,7 @@ async def connect_agent(self):
         except subprocess.CalledProcessError:
             raise StartFailed
 
-        dirs = subprocess.check_output(['gpgconf', '--list-dirs'])
+        dirs = subprocess.check_output(['gpgconf', '--list-dirs', '-o/dev/stdout'])
```

Comment:
Do you have a scenario where this happens and makes a difference?

Reply:
Sorry should have made my questions more clear. That basic idea is clear, I meant a realistic scenario that makes this worth it.

@marmarek what do you think, is checking stdout write error worth checking in every tool we invoke? I tend towards no.

Reply:
I prefer yes, mostly out of a desire for pedantic correctness and absolute robustness.  That said, another approach is to write a tool that uses `ptrace(2)` to work around this for *all* commands, by forcing a non-zero exit code if writing to `stdout` or `stderr` or reading from `stdin` fails.